src/tabs/presets/presets.js

```diff
@@ -335,13 +335,16 @@ presets.multipleSelectComponentScrollFix = function() {
         when the number of items 199+. More details here:
         https://github.com/wenzhixin/multiple-select/issues/552
     */
+   return new Promise((resolve) => {
     GUI.timeout_add('hack_fix_multipleselect_scroll', () => {
         this._selectCategory.multipleSelect('refresh');
         this._selectKeyword.multipleSelect('refresh');
         this._selectAuthor.multipleSelect('refresh');
         this._selectFirmwareVersion.multipleSelect('refresh');
         this._selectStatus.multipleSelect('refresh');
+        resolve();
     }, 100);
+   });
 };
```

Comment:
Good approach, but we are hooking into time base flakey event anyways 😄 Wondering if there are better events somewhere to hook in.

Reply:
Yeah unfortunately because of the bug in drop-down component we have to timout for 100ms...